src/test/details.test.js

```diff
@@ -1,16 +1,11 @@
 import React from 'react';
 import TestRenderer from 'react-test-renderer';
-import { Provider } from 'react-redux';
-import Details from '../pages/Details';
-import store from '../redux/configureStore';
+import countriesData from '../countriesData.json';
+import CountryDisplay from '../components/CountryDisplay';
 
```

Comment:
- [ ] As the previous reviewer mentioned some of the tests are not passing, kindly consider fixing this. The component `Countries` is misspelled on the `countries.test.js`, it should be countries with small letter `c` instead of `C`

![Screenshot_20220904_004039](https://user-images.githubusercontent.com/25227022/188290834-509c08de-eac6-4438-92f8-4af876899ec0.png)